.github/workflows/release.yaml

```diff
@@ -0,0 +1,118 @@
+name: Release Operator
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version'
+        required: true
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          ref: 'main'
+
+      - name: Set up Kubectl
+        uses: azure/setup-kubectl@v1
+
+      - name: Create and Push Tag
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.name 'github-actions'
+          git config --global user.email 'github-actions@github.com'
+          git tag ${{ github.event.inputs.version }}
+          git push origin ${{ github.event.inputs.version }}
+
+      - name: Log into registry quay.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
+        with:
+          registry: quay.io/ansible/
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build Operator Image
+        run: |
+          IMG=eda-server-operator:${{ github.event.inputs.version }} make docker-buildx
+          docker tag eda-server-operator:${{ github.event.inputs.version }} eda-server-operator:latest
+
+      - name: Push Operator Image
+        uses: redhat-actions/push-to-registry@v2.1.1
+        with:
+          image: eda-server-operator
+          tags: ${{ github.event.inputs.version }}
+          registry: quay.io/ansible/
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Push Operator Image
+        uses: redhat-actions/push-to-registry@v2.1.1
+        with:
+          image: eda-server-operator
+          tags: latest
+          registry: quay.io/ansible/
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Install operator-sdk
+        run: |
+          OPERATOR_SDK_VERSION=v1.23.0
+          curl -Lo operator-sdk "https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk_linux_amd64"
+          chmod +x operator-sdk
+          sudo mv operator-sdk /usr/local/bin/
```

Comment:
do we need this since the bundle target already call the operator-sdk target which does the same job ?
Also the operator sdk version in this task and the one defined in the Makefile is different. Is there a reason for that ?

Reply:
I have removed this task now that the typo is fixed, thanks!